powersimdata/scenario/create.py

```diff
@@ -15,6 +16,14 @@
 from powersimdata.scenario.state import State
 from powersimdata.utility import server_setup
 
+default_exported_methods = (
+    "create_scenario",
+    "get_bus_demand",
+    "print_scenario_info",
+    "set_builder",
+    "set_grid",
+)
```

Comment:
Do we need this tuple outside of the class? We should be able to move it in the constructor:
```
self.exported_methods = (
    "create_scenario",
    "get_bus_demand",
    "print_scenario_info",
    "set_builder",
    "set_grid",
)
```
No?

Reply:
I think you're right, as long it's not defined at class level so that it's not shared.

Reply:
Done.